test/sql/alter/alter_column/test_drop_not_null.test

```diff
@@ -0,0 +1,45 @@
+# name: test/sql/alter/not_null/test_drop_not_null.test
+# description: Test ALTER TABLE ALTER COLUMN column_name DROP NOT NULL
+# group: [alter_column]
+
+# Happy path
+
+statement ok
```

Comment:
Could you add `PRAGMA enable_verification` to all your tests please

Reply:
Sure. Will do that. Thank!